src/ng/directive/input.js

```diff
@@ -1647,7 +1647,8 @@ function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
       // IE11 doesn't set the el val correctly if the maxVal is greater than the element value
```

Comment:
I just noticed this comment should say "less than" (instead of "greater than"), right?